make-up/code/5163-mihaela-milosevska/src/app/models/mapping.ts

```diff
@@ -0,0 +1,45 @@
+import { Recipe, Ingredients} from "./client";
+import { RecipeResponse, IngredientsResponse } from "./server";
```

Comment:
The point of the mapping is to transform the data to a better format. In this case both formats are the same, so the transformation is in place. Still, the code is ok, +5 points